makefiles/platform/Darwin.mk

```diff
@@ -41,4 +41,5 @@ endif
 
 THEOS_SUDO_COMMAND ?= sudo
 _THEOS_PLATFORM_GET_LOGICAL_CORES := sysctl -n hw.ncpu
+_THEOS_PLATFORM_SWIFT ?= swift
```

Comment:
Is there a reason we're allowing reassignment here but not on a Linux host?

Reply:
Not particularly, though this is underscored so clients aren't expected to override it in the first place. I do see the value in reassigning it though, so I'd be happy to add a configurable non-underscored variant — but I might save that for a follow-up PR.